test/NuGet.Clients.Tests/NuGet.PackageManagement.VisualStudio.Test/Services/NuGetPackageSearchServiceTests.cs

```diff
@@ -275,6 +275,38 @@ public async Task GetPackageVersionsAsync_WhenIsTransitiveAndCacheIsNotPopulated
             }
         }
 
+        [Fact]
+        public void ClearFromCache_WhenCalled_PackageSearchMetadataMemoryCacheHasItemCleared()
```

Comment:
Nit: more specific naming? Maybe just:
```suggestion
        public void ClearFromCache_WhenPackageSearchMetadataMemoryCacheHasItem_ItemCleared()
```